CMakeLists.txt

```diff
@@ -337,3 +329,4 @@ install(CODE "set(SKIP_HEADERS_SUBDIR TRUE)")
 install(SCRIPT cmake/scripts/generate_config.cmake)
 install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(DIRECTORY licensing DESTINATION ${CMAKE_INSTALL_DOCDIR})
+install(FILES cmake/oneDPLWindowsIntelLLVMApply.cmake cmake/oneDPLWindowsIntelLLVMConfig.cmake DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR}/cmake/oneDPL/)
```

Comment:
According to https://cmake.org/cmake/help/v3.26/module/GNUInstallDirs.html, if `CMAKE_INSTALL_FULL_LIBDIR` is not an absolute path, then installation prefix will be prepended as with `CMAKE_INSTALL_LIBDIR`. So the outcome looks the same.
However, I suggest using `CMAKE_INSTALL_LIBDIR` to be aligned with the commands above.

Reply:
done